projects/plugins/jetpack/modules/shortcodes/slideshow.php

```diff
@@ -139,7 +139,7 @@ public function shortcode_callback( $attr ) {
 		$gallery = array();
 		foreach ( $attachments as $attachment ) {
 			$attachment_image_src   = wp_get_attachment_image_src( $attachment->ID, $attr['size'] );
-			$attachment_image_src   = $attachment_image_src[0]; // [url, width, height].
+			$attachment_image_src   = false !== $attachment_image_src ? $attachment_image_src[0] : ''; // [url, width, height].
```

Comment:
Could `$attachment_image_src` be true as well? Maybe this could also work:

```suggestion
			$attachment_image_src   = ! is_bool( $attachment_image_src ) ? $attachment_image_src[0] : ''; // [url, width, height].
```

Reply:
Based on https://developer.wordpress.org/reference/functions/wp_get_attachment_image_src/ return values are  `Array|false`

Reply:
Gotcha.